src/applications/ask-va/components/search/SearchControls.jsx

```diff
@@ -83,10 +83,11 @@ const SearchControls = props => {
               type="button"
               className="use-my-location-link"
             >
-              <i
+              <va-icon
+                size={4}
+                icon="null"
```

Comment:
This isn't a valid value for va-icon. Please check the [va-icon guidance](https://design.va.gov/storybook/?path=/docs/uswds-va-icon--docs) to determine which icon you need.

Reply:
Updated to near_me icon per guidance